webpack/envs/clientNovoProductionConfig.js

```diff
@@ -0,0 +1,235 @@
+// @ts-check
+
+const { basePath, env } = require("../utils/env")
+const { getCSSManifest } = require("../utils/getCSSManifest")
+const { HashedModuleIdsPlugin } = require("webpack")
+const { RetryChunkLoadPlugin } = require("webpack-retry-chunk-load-plugin")
+const path = require("path")
+const TerserPlugin = require("terser-webpack-plugin")
+const webpack = require("webpack")
+const WebpackManifestPlugin = require("webpack-manifest-plugin")
+const LoadablePlugin = require("@loadable/webpack-plugin")
+
+export const clientNovoProductionConfig = {
+  stats: "normal",
+  parallelism: 100,
+  mode: env.webpackDebug ? "development" : env.nodeEnv,
+  devtool: "source-map",
+  entry: {
+    "artsy-novo": [path.resolve(process.cwd(), "src/novo/src/client.tsx")],
+  },
```

Comment:
Same here, we can use smart merging to "extend" base artsy production config and update the `entry` key to point to new endpoint; no need to duplicate everything.

Reply:
See the previous comment about duplication.